docs/developer/release-process.md

```diff
@@ -55,32 +55,41 @@ The github action will create docker images and bundles.
 
 # How to release OperatorHub bundle
 
-Follow the process in: https://operatorhub.io/contribute
+OperatorHub has been split to two different community parts, one appearing for Kubernetes OLM installations and one for Openshift's integrated OperatorHub instance. We need to publish to both.
+It happens by creating a PR to two different repositories:
 
-The output of ``make bundle bundle-build`` should be valid if the development process was correct. Create the bundle which will be submitted in the PR with a correct version, for example for v1.10.0:
+https://github.com/redhat-openshift-ecosystem/community-operators-prod
+https://github.com/k8s-operatorhub/community-operators
 
-``make IMG=k8ssandra/cass-operator:v1.10.0 bundle bundle-build``
+A script has been created which will create the necessary packages, branch and a commit. The prerequisite is that those two must be checked out to same level directory as cass-operator so that they
+can be accessed with ``../community-operators/`` and ``../community-operators-prod``. Only users in the ``cass-operator-community/ci.yaml`` in those repository can submit updates. The commits
+must be signed, thus ensure your git properties are correctly set.
 
-Add the bundle directory as the input for the PR one submits for operatorhub.io
+The script is run by first checking out the correct tag in the ``cass-operator`` repository and then running:
+
+``scripts/release-community-olm.sh version`` 
+
+where ``version`` is without ``v``-prefix, for example ``1.11.0``. You need to manually push the created branch ``cass-operator-$VERSION`` and make a PR, the script will not do that part.
```

Comment:
No, it isn't as simple as it sounds. There are two obstacles as far as I can see. I don't know the fork's git name (so I can't do --set-upstream origin for example) since people checkout their repositories in different ways. Some use origin as the fork's name, some use origin for the upstream repo. The other reason (which I need to check if I documented..) is that not everyone can push them, only users in the allowed list are able to do that. Otherwise the RH's machine will reject it. The third is that the title of the PR has a structure which must be followed (and I'm not sure if I can open the PR with all those details).

If the user rights weren't restricted, I'd probably run this from a GHA, but that's not possible. Unless I use my own fork to do that.